src/app/components/CapStoryChart.tsx

```diff
@@ -55,7 +59,7 @@ export default function CapStoryChart({ rounds }: Props) {
     ];
   }, [rounds]);
 
-  return (
+  return showChart ? (
```

Comment:
In the latest version of recharts there seems to be a UI bug thats causing the chard legend to be miss-aligned. 
Quick-fixed by slightly delaying the rendering of the chart (humans wont notice)

Staying on older recharts version would have cause a cascade of other dependencies that could then not be updated (mainly react itself)